projectgenerator/gui/generate_project.py

```diff
@@ -91,11 +91,11 @@ def __init__(self, base_config, parent=None):
     def accepted(self):
         configuration = self.updated_configuration()
 
-        if not self.ili_file_line_edit.validator().validate(configuration.ilifile, 0)[0] == QValidator.Acceptable \
-                and not self.ili_models_line_edit.text():
-            self.txtStdout.setText(self.tr('Please set a valid INTERLIS file or model before creating the project.'))
-            self.ili_file_line_edit.setFocus()
-            return
+        if self.type_combo_box.currentData() == 'ili':
+            if not self.ili_file_line_edit.validator().validate(configuration.ilifile, 0)[0] == QValidator.Acceptable:
```

Comment:
We can also run without an ili file by providing a model name from a (remote or local) repository